dfastmi/gui/dialog_model.py

```diff
@@ -116,21 +116,11 @@ def qthreshold(self) -> float:
         """Get Qthreshold."""
         return self.section.getfloat("Qthreshold", 0.0)
 
-    @qthreshold.setter
-    def qthreshold(self, value: float):
-        """Set Qthreshold."""
-        self.section["Qthreshold"] = str(value)
-
     @property
     def ucritical(self) -> float:
```

Comment:
The viewmodel now only makes use of the readonly qthreshold and ucritical when loading a config, in all other cases it will use the view_model version of these properties.

And these propeties will only be update/set in the model when you get a configuration.

Reply:
yes this is expected.

Reply:
Discussed, this is correct.